core/src/main/java/com/netflix/conductor/core/events/ScriptEvaluator.java

```diff
@@ -25,19 +25,21 @@ private ScriptEvaluator() {
     }
 
     public static Boolean evalBool(String script, Object input) throws ScriptException {
-        Object ret = eval(script, input);
-
-        if (ret instanceof Boolean) {
-            return ((Boolean) ret);
-        } else if (ret instanceof Number) {
-            return ((Number) ret).doubleValue() > 0;
-        }
-        return false;
+        return toBoolean(eval(script, input));
     }
 
     public static Object eval(String script, Object input) throws ScriptException {
         Bindings bindings = engine.createBindings();
         bindings.put("$", input);
         return engine.eval(script, bindings);
     }
+
+    public static Boolean toBoolean(Object result) {
```

Comment:
Please add a javadoc for this method

Reply:
(side-note) It would be great to add javadocs for the other methods in this class as well

Reply:
Added javadocs.